### PR TITLE
Use InaSAFE's definition

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

**Comment by lucernae** [comment]
So quick. Much wow

File: src/realtime/ash/ash_hazard.py [hunk 1]
```diff
@@ -130,82 +159,49 @@ def copy_style(self):
...
+                extra_keyword_region['key']: self.region,
+                extra_keyword_volcano_alert_level['key']: self.alert_level,
+                extra_keyword_time_zone['key']: str(self.time_zone),
+                extra_keyword_volcano_latitude['key']: str(self.latitude),
+                extra_keyword_volcano_longitude['key']: str(self.longitude),
+                extra_keyword_volcano_event_id['key']: self.ash_id,
+                extra_keyword_volcano_eruption_event_time['key']: str(
+                    self.event_time),
+                extra_keyword_eruption_height['key']: str(self.eruption_height),
+                extra_keyword_volcano_height: str(self.vent_height),
```
Comment (lucernae) [inline src/realtime/ash/ash_hazard.py]: maybe typo here, you forgot the key

File: src/realtime/ash/ash_hazard.py [hunk 2]
```diff
@@ -130,82 +159,49 @@ def copy_style(self):
...
+                    u'medium': [2.0, 5.0],
+                    u'low': [0.1, 2.0]
+                }
+            }
+        }
+
         keywords = {
             'active_band': 1,
-            'hazard_category': u'single_event',
+            'hazard_category': hazard_category_single_event['key'],
```
Comment (lucernae) [inline src/realtime/ash/ash_hazard.py]: I thought the key like 'hazard_category' have definition too?

Comment (ismailsunni) [reply src/realtime/ash/ash_hazard.py]: Well, actually I had plan for that, and started working on it... But, my plan is too far (i.e. also put the same key in the metadata), and it needs much refactoring (which is exhausting but not seen). So, I decided to pause the work.... Then, for now, we need to use this way.

File: src/realtime/ash/ash_hazard.py (hunk 1 above)
Comment (ismailsunni) [reply src/realtime/ash/ash_hazard.py]: Ah yes, thanks.... Too many copy paste :D

**Pull request merged**